src/PrestaShopBundle/Service/Cache/Refresh.php

```diff
@@ -55,7 +55,7 @@ public function __construct($env = 'prod')
         $this->commands = [];
 
         require_once _PS_ROOT_DIR_.'/app/AppKernel.php';
-        $kernel = new \AppKernel($this->env, true);
+        $kernel = new \AppKernel($this->env, false);
```

Comment:
If defined, can't we use the constant `_PS_MODE_DEV_` instead of hardcoding a value ? (https://github.com/Shudrum/PrestaShop/blob/install-optimizations/admin-dev/index.php#L80)